com.github.devalien.workspaces.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "com.github.devalien.workspaces",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "command": "com.github.devalien.workspaces",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
Why does it need full host access?

Reply:
Yes, besically it is a command launcher and you might need access to different things on your filesystem. But the app by default does not launch any command, the commands are defined by the user and then launched by the user itself